docs/project_meltshiny/about.md

```diff
@@ -14,7 +14,7 @@ MeltShiny is a software application that automates the analysis and visualizatio
 - **Clients:**
   - [Dr. Brent Znosko](https://www.slu.edu/science-and-engineering/academics/chemistry/faculty/brent-znosko.php)
   - Sebastian Arteaga (Assistant)
-- **Current Tech Lead:** Sailikhita Pulijala [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/LikhitaPulijala)
```

Comment:
We want to make sure we continue giving credit to all our contributors. Please move Likhita to the Developers section and add "(alumni, prior tech lead)" to hers line, and edit Uday's line from "previous" to "prior". I think I want to move to that word across all the portfolio pages.